src/singletons.h

```diff
@@ -11,6 +11,12 @@
 #include <gtkmm.h>
 #include <string>
 
+#ifdef _WIN32
+#define HOME "AppData"
```

Comment:
Call the variable JUCI_HOME, there might be a variable already called HOME on some system. Also, what if running juci in MSYS2? Is the home directory then not stored in the HOME env? Might be better to make a static function (get_home_path() or something in juci::filesystem) that checks if getenv("HOME") returns something, if not, use getenv("AppData"), on Windows that is.